telemetry-attribute-utils/src/main/java/org/hypertrace/telemetry/attribute/utils/messaging/OtelMessagingAttributes.java

```diff
@@ -0,0 +1,21 @@
+package org.hypertrace.telemetry.attribute.utils.messaging;
+
+/**
+ * OTEL specific attributes for Messaging system
+ */
+public enum OtelMessagingAttributes {
+  MESSAGING_SYSTEM("messaging.system"),
+  MESSAGING_URL("messaging.url"),
+  RABBITMQ_MESSAGING_SYSTEM_VALUE("rabbitmq"),
+  RABBITMQ_ROUTING_KEY("rabbitmq.routing_key");
```

Comment:
Didn't find this in - https://github.com/open-telemetry/opentelemetry-specification/blob/master/semantic_conventions/trace/messaging.yaml

Reply:
Check here https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md